docs/references/backend/user/create-user.mdx

```diff
@@ -201,6 +201,20 @@ function createUser(params: CreateUserParams): Promise<User>
 
   ---
 
+  - `legalAcceptedAt?`
+  - `string`
```

Comment:
```suggestion
  - `Date`
```

Reply:
the backend api lists it as string or null, but it looks like the backend sdk has it listed as a `Date`
https://github.com/clerk/javascript/blob/main/packages/backend/src/api/endpoints/UserApi.ts#L78

Reply:
please apply this update to updateUser as well :)